extensions/wikia/PageHeader/PageTitle.class.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace PageHeader;
```

Comment:
Following the convention used in `PortableInfobox` extension, we have `namespace Wikia\CommunityHeader;`. We should unify, one way or the other.